omero/developers/Web/Deployment.rst

```diff
@@ -22,33 +22,33 @@ If you are using this option, you can go directly to the
 Installing OMERO.web
 --------------------
 
-You will need to follow the install instructions for OMERO.web,
-either as a standalone install, or as part of the OMERO.server install.
+From OMERO 5.6.0 release, the ``omero-web`` library supports Python 3 and
+can be installed via ``pip``. We recommend you use a virtual environment:
 
-If you want to connect to a remote server, you can add the server to the
-:property:`omero.web.server_list` and choose that server when you log in.
-You should enable :property:`omero.web.debug` and start a lightweight
-development Web server on your local machine.
+    $ python3 -m venv py3_venv
+    $ source py3_venv/bin/activate
 
-.. note:: Since OMERO 5.2, the OMERO.web framework no longer bundles
-    a copy of the Django package, instead manual installation of
-    the Django dependency is required. It is highly recommended to use
-    `Django 1.8`_ (LTS) which requires Python 2.7. For more information
-    see :ref:`python-requirements` on the
-    :doc:`/sysadmins/version-requirements` page.
+    $ pip install "omero-web==v5.6.dev5"
+
+    Successfully installed Django-1.11.26 Pillow-6.2.1 django-pipeline-1.6.14
+    future-0.18.2 gunicorn-20.0.0 omero-marshal-0.6.3 omero-py-5.6.dev5
+    omero-web-5.6.dev5 pytz-2019.3 zeroc-ice-3.6.5
 
 Using the lightweight development server
 ----------------------------------------
 
 All that is required to use the Django lightweight development server
 is to set the :property:`omero.web.application_server` configuration option,
-turn :property:`omero.web.debug` on and start the server up:
+turn :property:`omero.web.debug` on.
+We also configure a remote server to connect to, and then
+start the server up.
 
 ::
 
-    $ bin/omero config set omero.web.application_server development
-    $ bin/omero config set omero.web.debug True
-    $ bin/omero web start
+    $ omero config set omero.web.application_server development
```

Comment:
This will need the addition of OMERODIR in a follow-up.

Reply:
See https://github.com/ome/ome-documentation/pull/2037